vars/slackUtils.groovy

```diff
@@ -66,6 +70,8 @@ def sendMsg(parameters = [:]) {
 
     slackChannel.each { channel ->
         slackSend(
+            teamDomain: slackTeamDomain,
```

Comment:
Are we sure these can safely be `null`?

Can you test a simple pipeline job that calls this:

```
slackUtils.sendMsg(channel: #some-channel, msg: "test")
```

Just to make sure that all of this is backward-compatible and still sends to ansible slack by default?

Reply:
I set the default value as "ansible" as you can see here: https://github.com/RedHatInsights/insights-pipeline-lib/pull/102/files#diff-8eae8c62c616ee22fdbded03556311e8R48

Reply:
True, ok..